src/modules/bot/commands/questions/questions-interaction.collector.ts

```diff
@@ -155,12 +169,44 @@ export class QuestionsInteractionCollector {
       }, answerTimeInMilliseconds)
     }
 
+    this.stopButton.setDisabled(false)
+    const buttonRow = new ActionRowBuilder<MessageActionRowComponentBuilder>().addComponents(
+      this.watchButton,
+      this.listenButton,
+      this.stopButton
+    )
+
     await interaction.reply({
       content: `Şu sorunun yanıtı ses kanalında oynatılıyor: **${selectedQuestion.title}** \n\n${
         !selectedQuestion.endTime.minute || !selectedQuestion.endTime.second
           ? '🚨 Uyarı: Bu sorunun bitiş süresi bulunamadı. Ses video bitene kadar oynamaya devam edecek.'
           : ''
       }`,
+      ephemeral: true,
+      components: [buttonRow]
+    })
+  }
+
+  private async stopButtonInteraction(
+    interaction: ButtonInteraction
+  ): Promise<InteractionResponse<boolean>> {
```

Comment:
This interaction just sending replies. So we don't need to return anything from here. Please change this as `Promise<void>`.